obvs/patchscope.py

```diff
@@ -168,8 +171,10 @@ def __init__(self, source: SourceContext, target: TargetContext) -> None:
 
         self.tokenizer = self.source_model.tokenizer
 
-        self.MODEL_SOURCE, self.LAYER_SOURCE = self.get_model_specifics(self.source.model_name)
-        self.MODEL_TARGET, self.LAYER_TARGET = self.get_model_specifics(self.target.model_name)
+        self.MODEL_SOURCE, self.LAYER_SOURCE, self.ATTN_SOURCE, self.HEAD_SOURCE = \
```

Comment:
nit: to be compliant with PEP8, can you change the variable names to `_lowercase`? Like `self._model_source`?

Reply:
Agreed -> will change to lowercase